src/main/webapp/app/entities/result.model.ts

```diff
@@ -45,4 +45,13 @@ export class Result implements BaseEntity {
     public static isManualResult(that: Result): boolean {
         return that.assessmentType === AssessmentType.MANUAL || that.assessmentType === AssessmentType.SEMI_AUTOMATIC;
     }
+
+    /**
+     * Checks whether the result is generated by an AI.
+     *
+     * @return true if the result is an automatic AI result
+     */
+    public static isAutomaticAIResult(that: Result): boolean {
```

Comment:
Same here? „That“ seems rather non-descript?

Reply:
In fact, would you say it makes sense to refactor other methods as well?

Reply:
Doesn't have to be as part of this PR - I think this can be done in a follow up